org.oasis.xdita/dtd/lw-map.mod

```diff
@@ -93,77 +92,51 @@ PUBLIC "-//OASIS//ELEMENTS LIGHTWEIGHT DITA Map//EN"
-              %reference-content;
-              keys
-                        CDATA
-                                  #REQUIRED
-              processing-role
-                        CDATA       #FIXED      'resource-only'
-              outputclass  CDATA          #IMPLIED
-              class CDATA "+ map/topicref mapgroup-d/keydef ">
-              
-<!--                    LONG NAME: Link text-->
-<!ELEMENT linktext     (#PCDATA | %ph;)* >
-<!ATTLIST linktext
-             %localization;
-             outputclass  CDATA          #IMPLIED
-             class CDATA "- map/linktext ">
-             
 <!--                    LONG NAME: Map  -->
 <!ELEMENT map		(topicmeta?, (topicref | keydef)*)  >
 <!ATTLIST map
              id       ID          #IMPLIED
              xmlns:ditaarch CDATA #FIXED "http://dita.oasis-open.org/architecture/2005/"
-	         ditaarch:DITAArchVersion CDATA "1.3"
-             domains    CDATA                    "&xdita-constraint; &included-domains;"
+	         ditaarch:DITAArchVersion CDATA "2.0"
+             specializations    CDATA                    "&included-domains;"
              %localization;
              outputclass  CDATA          #IMPLIED
              class CDATA "- map/map ">
 
-<!--                    LONG NAME: Navigation title -->
-<!ELEMENT navtitle (#PCDATA|%ph;)* >
-<!ATTLIST navtitle
-             %localization;
-             outputclass  CDATA          #IMPLIED
-             class CDATA "- topic/navtitle "> 
 
 <!--                    LONG NAME: Metadata-->
-<!ELEMENT topicmeta     (navtitle?, linktext?, data*) >
+<!ELEMENT topicmeta     (navtitle?, keytext?, data*) >
 <!ATTLIST topicmeta
              %localization;
-             class CDATA "- map/topicmeta ">            
+             class CDATA "- map/topicmeta ">
              
-<!--                    LONG NAME: Topic or Map Reference  -->
-<!ELEMENT topicref	(topicmeta?, topicref*)        >
-<!ATTLIST topicref
+<!--                    LONG NAME: Key text                        -->
+<!ELEMENT keytext         (#PCDATA | %ph;)*  >
+<!ATTLIST keytext
+             %localization;
+             class CDATA "- map/keytext ">            
+
+<!--                    LONG NAME: Navigation title -->
+<!ELEMENT navtitle (#PCDATA|%ph;)* >
+<!ATTLIST navtitle
              %localization;
-             locktitle CDATA      			 #FIXED 'yes'
-	         %reuse;
-             %filters;
-             %reference-content;
-	         %control-variables;
-             %variable-links;
-             processing-role
-                        CDATA       #FIXED      'resource-only'
              outputclass  CDATA          #IMPLIED
-             class CDATA "- map/topicref ">
+             class CDATA "+ topic/titlealt alternativeTitles-d/navtitle ">
 
-<!--                    LONG NAME: Alternative content  -->
-<!ELEMENT alt           (#PCDATA|%ph;|%data;)*        >
-<!ATTLIST alt
+<!--<!-\-                    LONG NAME: Link text-\->

```

Comment:
I assume this element is being deleted? If so it should be removed rather than commented out